fireblocks_sdk/api_types.py

```diff
@@ -369,31 +368,141 @@ class TimePeriod(str, Enum):
     DAY = "DAY"
     WEEK = "WEEK"
 
+class StellarRippleCreateParams:
+    def __init__(
+        self,
+        issuerAddress: Optional[str] = None
+    ):
+        self.issuerAddress = issuerAddress
 
-class IssueTokenRequest:
-    symbol: str
-    name: str
-    blockchain_id: str
-    eth_contract_address: Optional[str]
-    issuer_address: Optional[str]
-    decimals: int
+    def to_dict(self):
+        return convert_class_to_dict(self.__dict__)
 
-    def serialize(self) -> dict:
-        obj = {
-            'symbol': self.symbol,
-            'name': self.name,
-            'blockchainId': self.blockchain_id,
-            'decimals': self.decimals,
-        }
+class Parameter:
```

Comment:
When going over this, I think you should use dataclass for these types this will reduce a lot of the code you wrote
https://docs.python.org/3/library/dataclasses.html